donate.html

```diff
@@ -65,12 +65,86 @@
+            <div class="large-spark purp-spark" style="top: -35px; right: -40px;"></div>
+            <div class="small-spark pink-spark" style="top: 150px; right: -25px;"></div>
+            <div class="med-spark blue-spark" style="top: 70px; right: -110px;"></div>
+          </div>
+        </span>
+      </h2>
+        <img src="img/technica-logo.png" width="270" height="60">
+        <div class="b1"> <button>Donate</button> </div>
+      </div>
+    </div>
+  </div>
+
+  <!-- Our Story  -->
+  <div class="donate-story">
+    <div class="container">
+      <!-- Header -->
+      <div class="story-header">
+        <h1>Our Story</h1>
+      </div>
 
-</div>
+      <!-- Body -->
+      <div class="row">
+        <div class="col-md-6">
+          <div class="story-body">
+            <p> Technica was founded in 2015 by empowered students at the University of Maryland who recognized the need for an inclusive all-women & non-binary space in the hackathon scene. </p>
+            <p> As the world’s largest all-women & non-binary hackathon, we are often asked how one can support our cause. With time constraints and distance in mind, we’ve created a way for individuals to offer support: financial donations. </p>
+            <p> We know money can be tight, so any and all donations are greatly appreciated. <b>Pitching in a few bucks towards our cause means we can continue offering the highest quality experience for our hackers.</b> </p>
+          </div>
+        </div>
+        <div class="col-md-6">
+          <div class="story-image">
+            <img src="img/2019_img.jpg" class="center-block" width="350" height="200">
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <!-- Donor Promise -->
+  <div class="donate-promise">
+    <div class="container">
+      <!-- Header -->
+        <div class="promise-header">
+          <div class="row">
+            <div class="col-md-6"> </div>
+            <div class="col-md-6"> <h1>Our Donor Promise</h1></div>
+          </div>
+        </div>
+
+      <!-- Body -->
+      <div class="row">
+        <div class="col-md-6">
+          <div class="promise-image">
+              <img src="img/2017_img.jpg" class="center-block" width="350" height="200">
+          </div>
+        </div>
+        <div class="col-md-6">
+          <div class="promise-body">
+            <p> <b> We promise to respect your donation by only using donor funds to support our mission of reaching gender equity in tech. </b> The most high cost ways we spend funds are to rent out our hackathon venue space, providing meals to our hackers all weekend long, and ensuring safety in the form of meeting fire marshal regulations and event health personnel</p>
+            <p> <b> Additionally, Technica is a 501(c)(3) public charity, so your donations are tax-deductible. </b></p>
+          </div>
+        </div>
+      </div>
+    </div>
+  <!-- jules -->
```

Comment:
move this jules comment to below your code so git registers your code as above her code when we merge it all together